packages/requisitions/src/RequestRequisition/DetailView/Toolbar/ToolbarActions.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {
+  Autocomplete,
+  InputWithLabelRow,
+  useTranslation,
+  useConfirmationModal,
+} from '@openmsupply-client/common';
+import { useRequestFields, useIsRequestDisabled } from '../../api';
+
+const months = [1, 2, 3, 4, 5, 6];
+
+export const ToolbarActions = () => {
+  const t = useTranslation('replenishment');
+  const isDisabled = useIsRequestDisabled();
+  const { minMonthsOfStock, update } = useRequestFields('minMonthsOfStock');
+  const getConfirmation = useConfirmationModal({
+    title: t('heading.are-you-sure'),
+    message: t('messages.changing-min-mos'),
+  });
+
+  return (
+    <InputWithLabelRow
```

Comment:
I also had a look at using the filter button which opens a small popover with these actions. It was OK, but I think I like this better 🤔